ClubMembersSortedCF/src/members/v1/ZipCodeValidator.java

```diff
@@ -0,0 +1,7 @@
+package members.v1;
+
+public class ZipCodeValidator {
+    boolean isValidZipCode(String zipCode){
+        return zipCode.matches("[1-9]\\d{3}");
```

Comment:
smart validator

Reply:
of course, you could implement a generic `Validator<T>` interface here where (with obviously replacing `T` with integer). not an issue, just +1 more opportunity to practice basic OOP